### PR TITLE
fix LINT warnings mostly renames

### PR DESCRIPTION
**Pull request by jpdup**
Simple rename of some missspelled variables

### CODE REVIEW

**Comment by jpdup** [comment]
@clovett can you approved this PR approved?

File: Source/WPF/MyMoney/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -46,9 +45,9 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: NeutralResourcesLanguage("")]
+[assembly: NeutralResourcesLanguageAttribute("")]
```
Comment (clovett) [inline Source/WPF/MyMoney/Properties/AssemblyInfo.cs:48]: Including the "Attribute" suffix in custom attributes is not required.

File: Source/WPF/MyMoney/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -46,9 +45,9 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: NeutralResourcesLanguage("")]
+[assembly: NeutralResourcesLanguageAttribute("")]
 
 [assembly: InternalsVisibleTo(@"MyMoneyTests, PublicKey=002400000480000094000000060200000024000052534131000400000100010075BA8B492C6DFC7D931414074F83ED4C2277EB06E3D2F0AC39A0EFDA3F4333436CC2E09BDCF70EEFD440487708441AA7B64B43D8147E1002B3D0754720C7E277214FFC3E9B9472FBC12BCAE89E9DABF7C13057B54DEC078896FB800C694907FAFAB201E8707CACC55A169D35D0CE9E1F47EA61AF75DD30BB362B22DCC1EBD7AF")]
 
-[assembly: SupportedOSPlatform("windows6.1")]
```
Comment (clovett) [inline Source/WPF/MyMoney/Properties/AssemblyInfo.cs:53]: why are we removing this?

**Review by clovett — CHANGES_REQUESTED** [review]
looks great, nice fix in FillinMissingUnitPrices.  Just a couple comments/questions

File: Source/WPF/MyMoney/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (jpdup) [reply Source/WPF/MyMoney/Properties/AssemblyInfo.cs:53]: I agree that for now this should not be included.
I find it strange that the lastest tool are altering these files, to be researched later

File: Source/WPF/MyMoney/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (jpdup) [reply Source/WPF/MyMoney/Properties/AssemblyInfo.cs:48]: This was altered by the build tools, it was emitting a build warning

Comment (clovett) [reply Source/WPF/MyMoney/Properties/AssemblyInfo.cs:48]: oh, weird, good to know thanks.

**Review by clovett — APPROVED** [review]

File: Source/WPF/MyMoney/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (clovett) [reply Source/WPF/MyMoney/Properties/AssemblyInfo.cs:53]: Oh, interesting it actually fails the build with this in there, perhaps it was in .NET 5.0 but they removed it in later versions of .NET ?

Comment (clovett) [reply Source/WPF/MyMoney/Properties/AssemblyInfo.cs:53]: But why does the Visual Studio build work fine on my machine...

Comment (jpdup) [reply Source/WPF/MyMoney/Properties/AssemblyInfo.cs:53]: How do you want to proceed? 
My vote is 
- Remove that line as suggested by the LINT tool
- Confirm that you can load and build correctly on your machine. 
- merge the PR

Comment (clovett) [reply Source/WPF/MyMoney/Properties/AssemblyInfo.cs:53]: Sounds good, I have verified I can build without the SupportedOSPlatform attribute and the app runs.  What bugs me is that the master branch also builds find without the changes in this PR, yet somehow your PR causes SupportedOSPlatform attribute to become a problem and I can't see what it is in your PR that is causing that and I don't like things I can't explain, they usually bite us in the ass later... but I'm willing to remove the line for now and merge the PR.

Comment (jpdup) [reply Source/WPF/MyMoney/Properties/AssemblyInfo.cs:53]: This is the only hint:

`The type or namespace name 'SupportedOSPlatformAttribute' could not be found (are you missing a using directive or an assembly reference?)`

**Pull request merged**